dapr_agents/storage/vectorstores/__init__.py

```diff
@@ -1,3 +1,5 @@
 from .base import VectorStoreBase
 from .chroma import ChromaVectorStore
 from .postgres import PostgresVectorStore
+
+__all__ = ["VectorStoreBase", "ChromaVectorStore", "PostgresVectorStore"]
```

Comment:
doesn't this mean when someone does a piip install on our project that they then bring in all of these dependencies? Do we want that? What if they dont use vector stores at all?